src/main/com/yetanalytics/lrs/util.cljc

```diff
@@ -14,3 +14,23 @@
        :cljs (.stringify js/JSON (clj->js x) nil 2))
     #?(:clj (json/generate-string x)
        :cljs (.stringify js/JSON (clj->js x)))))
+
+;; a la https://stackoverflow.com/questions/18735665/how-can-i-get-the-positions-of-regex-matches-in-clojurescript
+#?(:cljs
+   (defn regex-modifiers
+     "Returns the modifiers of a regex, concatenated as a string."
+     [re]
+     (str (if (.-multiline re) "m")
+          (if (.-ignoreCase re) "i"))))
```

Comment:
Missing `else` branches here.

Reply:
Single branch `if` is functionally identical to `when`, but sure we can change it for clarity